readme.md

```diff
@@ -1,6 +1,6 @@
 # Frontend practice with catalog page using grid
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog_grid/)
+- [DEMO LINK](https://KaterynaZhokheieva.github.io/layout_catalog_grid/)
```

Comment:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/38065883/175106486-d2c4f0b0-0e4f-440f-9583-c5f0f7cfe225.png">
align cards to the center